BUILDING.md

```diff
@@ -30,6 +30,21 @@ To build with only the TinkerPop tests:
 mvn clean install -Dtest.skip.tp=false -DskipTests=true
 ```
 
+To build and run a docker image with JanusGraph and Gremlin Server, configured
+to run the berkeleyje backend with an embedded ElasticSearch instance:
+
+```bash
+mvn clean package -Pjanusgraph-release -Dgpg.skip=true -DskipTests=true
+cd janusgraph-dist && mvn install -Pjanusgraph-docker -DskipTests=true docker:build
+docker run -d -p 8182:8182 --name janusgraph janusgraph/server:latest
+```
+
+To connect to the server in the same container on the console:
+
+```bash
+docker exec -i -t janusgraph /var/janusgraph/bin/gremlin.sh
+```
+
```

Comment:
You should continue this out with some Gremlin Console interaction. The BerkeleyJE backend only allows a single connection, so the users cannot do something like `graph = JanusGraphFactory.open('conf/janusgraph-berkeleyje-es.properties')` as described in the [Getting Started doc](http://docs.janusgraph.org/0.1.0-SNAPSHOT/getting-started.html#_loading_the_graph_of_the_gods_into_janusgraph).

They would need to configure a remote connection through the Gremlin Server.

```
gremlin> :remote connect tinkerpop.server conf/remote.yaml
==>Configured localhost/127.0.0.1:8182
gremlin> :remote console
==>All scripts will now be sent to Gremlin Server - [localhost/127.0.0.1:8182] - type ':remote console' to return to local mode
gremlin> GraphOfTheGodsFactory.load(graph)
==>null
gremlin> g = graph.traversal()
==>graphtraversalsource[standardjanusgraph[berkeleyje:db/berkeley], standard]
```